### PR TITLE
operator: Add no hugepages test

### PR DESCRIPTION
**Pull request by sebrandon1**
Adds an `operator` suite test for making sure they are not using hugepages in their pods that utilizes the existing `pod.HasHugepages()` function.

Note, I have left the test case as fully `Optional` for now until we discuss Mandatory or Optional as a group.

### CODE REVIEW

**Review by greyerof — APPROVED** [review]

File: tests/operator/suite.go [hunk 1]
```diff
@@ -123,6 +123,13 @@ func LoadChecks() {
 			testOperatorContainersReadOnlyFilesystem(c, &env)
 			return nil
 		}))
+
+	checksGroup.Add(checksdb.NewCheck(identifiers.GetTestIDAndLabels(identifiers.TestOperatorPodsNoHugepages)).
+		WithSkipCheckFn(testhelper.GetNoOperatorsSkipFn(&env)).
```
[the remark this answers is not in the document]
Comment (sebrandon1) [reply tests/operator/suite.go:128]: Good idea, I can add that.

[the remark this answers is not in the document]
Comment (sebrandon1) [reply tests/operator/suite.go]: Done in https://github.com/redhat-best-practices-for-k8s/certsuite/commit/417d60d5864e4cd1ba154e23959e6a8e2510414a

File: tests/identifiers/identifiers.go [hunk 2]
```diff
@@ -1047,6 +1048,22 @@ that Node's kernel may not have the same hacks.'`,
 		},
 		TagCommon)
 
+	TestOperatorPodsNoHugepages = AddCatalogEntry(
+		"pods-no-hugepages",
+		common.OperatorTestKey,
+		`Tests that check that the pods do not have hugepages enabled.`,
```
Comment (jmontesi) [inline tests/identifiers/identifiers.go:1054]:
```suggestion
		`Tests that the pods do not have hugepages enabled.`,
```

**Review by jmontesi — APPROVED** [review]

File: tests/identifiers/identifiers.go (hunk 2 above)
Comment (sebrandon1) [reply tests/identifiers/identifiers.go]: Done.

**Pull request merged**